src/anemoi/training/config/stretched.yaml

```diff
@@ -0,0 +1,36 @@
+defaults:
+- data: zarr
+- dataloader: native_grid
+- diagnostics: evaluation
+- hardware: example
+- graph: stretched_grid
+- model: gnn
+- training: default
+- _self_
+
+
+### This file is for local experimentation.
+##  When you commit your changes, assign the new features and keywords
+##  to the correct defaults.
+# For example to change from default GPU count:
+# hardware:
+#   num_gpus_per_node: 1
+
+dataloader:
+  dataset:
+    cutout:
+      - dataset: ${hardware.paths.data}/${hardware.files.dataset}
+        thinning: ???
+      - dataset: ${hardware.paths.data}/${hardware.files.forcing_dataset}
+    adjust: all
+    min_distance_km: ???
+#training:
+#  loss_scaling:
+#    spatial:
+#      _target_: anemoi.training.data.scaling.StretchedGridCutoutAreaWeights
+#      target_nodes: ${graph.data}
```

Comment:
With the way ReweightedGraphAttribute is currently set up you need to add
node_attribute: area_weight
scaled_attribute: cutout_mask